tutorials/creating-interiors/export-blender.md

```diff
@@ -0,0 +1,9 @@
+# Export from Blender
+To export you just need to select every item you want to export and open Sollumz tool and press export do note that 
+you can only export one thing at a time. Note that the object need to be visible to be exported so make sure that 
+everything you want to export is not hidden.
```

Comment:
> do note that you can only export one thing at a time
This is not correct, Sollumz will export every selected item. Don't have to do it one by one.